src/components/App/index.tsx

```diff
@@ -53,6 +57,8 @@ export default class App extends Component<Props, State> {
         window.STATE = this.state;
       };
     }
+
+    import('./client-api').then(m => m.exposeAPI(this));
```

Comment:
You probably need to give this a `webpackChunkName` as above.

Reply:
And make sure it's cached offline https://github.com/GoogleChromeLabs/squoosh/blob/master/src/sw/util.ts#L58.

Reply:
`webpackChunkName` doesn't appear to be working. I think it needs to come before the string. See the other examples in this file.